libs/client-api/src/http.rs

```diff
@@ -627,16 +628,26 @@ impl Client {
     AppResponse::<()>::from_response(resp).await?.into_error()
   }
 
+  pub async fn get_workspaces(&self) -> Result<Vec<AFWorkspace>, AppResponseError> {
+    self
+      .get_workspaces_opt(QueryWorkspaceParam::default())
+      .await
+  }
+
   #[instrument(level = "info", skip_all, err)]
-  pub async fn get_workspaces(&self) -> Result<AFWorkspaces, AppResponseError> {
```

Comment:
Is it compatible with old api endpoint?

Reply:
yes it is backward compatible. frontend appflowy have to change from using `AFWorkspaces` to `Vec<Workspace>` when update, but that's just rust structure, the http endpoint behaviour is same for old requests